### ISSUE TITLE
Accessibility: "People" & "Bounties" Tabs pattern issues in Bounties Workspace pages

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1234]
# Accessibility: "People" & "Bounties" Tabs missing some properties, states, and keyboard behaviors of a Tabs pattern control

## Issue
The "People" & "Bounties" Tabs do not follow the established design pattern for a Tablist.

## Expected Behavior
- [WCAG 2.2 AA 4.1.2 Name, Role, Value](https://www.w3.org/TR/wai-aria-1.2/#introstates) requires that programmatically determined states and properties are communicated to screen reader users.
- [WAI - ARIA APG Tabs Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/).

- Screen reader users expect to hear the correct tab role announced along with the selected state of the current tab so they know which tab is currently active.
- Keyboard users expect to encounter the [expected keyboard interaction](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/#keyboardinteraction) 
## Acceptance Criteria
- [ ] The element that serves as the container for the set of tabs has `role="tablist"`.
- [ ] Each element that serves as a tab has `role="tab"` and is contained within the element with `role="tablist"`.
- [ ] Each element that contains the content panel for a tab has `role="tabpanel"`.
- [ ] If the tab list has a visible label, the element with `role="tablist"` has `aria-labelledby` set to a value that refers to the labelling element. Otherwise, the `tablist` element has a label provided by `aria-label`.
- [ ] Each element with `role="tab"` has the property `aria-controls` referring to its associated `tabpanel` element.
- [ ] The active `tab` element has the state `aria-selected` set to `true` and all other `tab` elements have it set to `false`.
- [ ] Each element with `role="tabpanel"` has the property `aria-labelledby` referring to its associated tab element.
- [ ] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. - ensure the correct roles and selected tab are announced.
- [ ] Test with a keyboard to ensure the pattern follows the [expected keyboard interaction](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/#keyboardinteraction) 
-  [ ] Verify in Chrome and Firefox with and without a mobile viewport enabled to ensure visual look and feel is unchanged (with exception of unselected Tab text contrast which should be 4.5:1 against the background).

_Linked to tracking issue: #1233_
